packages/gestalt/src/Link.css

```diff
@@ -9,15 +9,17 @@
   text-decoration: underline;
 }
 
-.inlineBlock {
-  composes: inlineBlock from "./Layout.css";
+.inlineFlex {
+  composes: itemsCenter from "./Layout.css";
+  composes: justifyCenter from "./Layout.css";
+  display: inline-flex;
```

Comment:
Rather than declare this here, we should follow the existing pattern of establishing this style in Layout.css and composing here

Reply:
DONE